src/enums/filingStatus.ts

```diff
@@ -1,11 +1,15 @@
 export enum FilingStatus {
+  CANCELLED = 'CANCELLED',
   COMPLETED = 'COMPLETED',
+  CORRECTED = 'CORRECTED',
   DRAFT = 'DRAFT',
-  EPOCH = 'EPOCH',
-  ERROR = 'ERROR',
-  NEW = 'NEW',
+  DELETED = 'DELETED',
+  EPOCH = 'EPOCH', // soon to be obsolete
+  ERROR = 'ERROR', // soon to be obsolete
+  NEW = 'NEW', // for todo tasks only
   PAID = 'PAID',
   PENDING = 'PENDING',
-  PENDING_ALTERATION = 'PENDING_ALTERATION',
-  PENDING_CORRECTION = 'PENDING_CORRECTION',
+  PENDING_ALTERATION = 'PENDING_ALTERATION', // soon to be obsolete
+  PENDING_CORRECTION = 'PENDING_CORRECTION', // soon to be obsolete
+  WITHDRAWN = 'WITHDRAWN'
```

Comment:
I added missing types according to:

![filing-states](https://user-images.githubusercontent.com/10002889/131911259-3e2320a8-3d64-486a-b75e-ef8bb73774e5.png)